src/impl/device/opticalSensor.cpp

```diff
@@ -0,0 +1,55 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+#include "okapi/impl/device/opticalSensor.hpp"
+
+namespace okapi {
+OpticalSensor::OpticalSensor(std::uint8_t iport) {
+  port = iport;
+  pros::c::optical_disable_gesture(port);
```

Comment:
Why do you disable gestures?

Reply:
My understanding is the device is either in color-detection mode or gesture mode. It cannot do both, at least based on how I read the V5 code documentation. Figured defaulting to color-detection mode was a reasonable choice.